params/default.go

```diff
@@ -14,6 +15,51 @@ func DefaultNetwork() Network {
 }
 
 func init() {
+	// check if custom network option set
+	if customNet, ok := os.LookupEnv("CELESTIA_CUSTOM_GENESIS"); ok {
+		fmt.Print("\n\nWARNING: Celestia custom network specified. Only use this option if the node is " +
+			"freshly created and initialized.\n**DO NOT** run a custom network over an already-existing node " +
+			"store!\n\n")
```

Comment:
Is this printed out after the node is already running?

Reply:
Yes.

Reply:
Seems like a useless message then, since it could have bad effects on your already-existing setup. Is there some way to detect that then abort with this message instead?

Reply:
I can only really do this check on `Init` not on `Start` because `Start` will only ever run over a store that already exists (so we can't really fail out in `Start` if the store exists bc it *has* to exist in order for node to start). 

I will implement the check for `Init` but I think since this isn't a flag provided to users and instead is an environment variable, it should be fine.